Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:2.7.11-alpine
+
+RUN pip install requests
+
+ADD . /src
+
+# UDP
+EXPOSE 1812
```

Comment:
Guessing without more research I would say it has to do with the address or
port detected in recvfrom.

On Wednesday, March 30, 2016, Aren Sandersen <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/foxpass/foxpass-radius-proxy/pull/1#discussion_r57965348>
> :
>
> > @@ -0,0 +1,10 @@
> > +FROM python:2.7.11-alpine
> > +
> > +RUN pip install requests
> > +
> > +ADD . /src
> > +
> > +# UDP
> > +EXPOSE 1812
>
> I played with this a bit. It definitely needs the "/UDP" suffix. But I
> still could not get the networking to work without the "--net host"
> argument. Must be missing something.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/foxpass/foxpass-radius-proxy/pull/1/files/8225d7ce828d6d9af95f4bfefa238d706ffb561d#r57965348>
>